pkgs/development/python-modules/fido2/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchFromGitHub, six, cryptography }:
+
+buildPythonPackage rec {
+  pname = "fido2";
+  version = "0.3.0";
+  name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.